### PR TITLE
Tc1 display event time

### PR DESCRIPTION
**Pull request by Brianna-Flo**
## Description

<what this pull request is doing>

- add a prisma modal to store calendar events
- create an endpoint to add a calendar event and connect it to a user
- display a link to the google calendar event if upcoming for a given recipe
- Refetch recipes to display link
- Do not show past events, only upcoming calendar events

<what will be done in later PRs and not included here>

- further styling of the meal card component

## Milestones
<the milestones or stories/features that this works towards>

- tc1, user interactivity

## Resources
<links to tutorials, code snippets, inspirations>
<if you took or translated specific code from the internet, please call it out here!>
https://www.prisma.io/docs/orm/prisma-schema/data-model/relations

## Test Plan
<insert images or gifs of feature>
<otherwise, say how code was tested if not UI facing>
<any edge cases you might have specifically tested>
Screenshot 2025-07-22 at 11 54 24 PM

### CODE REVIEW

File: backend/routes/calendarRoutes.ts [hunk 1]
```diff
@@ -15,6 +15,35 @@ const SHOPPING_TIME = 60;
...
+        },
+      });
+      if (!newEvent) {
+        return res
+          .status(400)
+          .send("Error, failed to add the event to database");
+      }
+      res.json(newEvent);
+    } catch (error) {
+      res.status(500).send("Server Error");
```
Comment (paulshaoyuqiao) [inline backend/routes/calendarRoutes.ts:42]: do we want to log what kind of error it is?
also show error message

**Review by paulshaoyuqiao — APPROVED** [review]
